### ISSUE TITLE
#geocode with an unknown address does not return the `default_stub`'s coordinates

### ISSUE THREAD

**Issue by Stex** [alexreisner/geocoder#1197]
In our application, we have an `Address` model which uses `geocoded_by`.

In a spec, we wanted to test this with a non-stubbed query which should result in the default response we set up. `#geocode` actually sets the correct `latitude` and `longitude` in the record, but returns `nil` instead of `[lat, long]`.

Why we actually need the result of `#geocode`: We validate that the data our users enter can actually be processed by - in our case - google's geocoding service, so if `#geocode` returns `nil`, the user entered invalid data.

We solved this for now by adding a `coordinates` key to the default stub, but this is neither reflected in the `README` nor should it be necessary as the hash already contains `latitude` and `longitude`.

Am I doing something wrong here or is this a bug in the stubbing system?

```ruby
Geocoder::Lookup::Test.set_default_stub(
  [
    {
      'latitude'          => 16.7665887,
      'longitude'         => -3.0025615,
      # Without the following line, `my_address.geocode` returns nil, but sets the
      # correct lat/long in the object anyway.
      # 'coordinates'       => [16.7665887, -3.0025615],
      'postal_code'       => nil,
      'city'              => 'Timbuktu',
      'state'             => 'Tombouctou',
      'state_code'        => nil,
      'country'           => 'Mali',
      'country_code'      => 'ML',
      'address'           => 'Rue de Chemnitz 1, Timbuktu, Mali',
      'street_address'    => 'Rue de Chemnitz 1',
      'route'             => 'Rue de Chemnitz',
      'street_number'     => 1
    }
  ]
)
```

**Comment by alexreisner** [comment]
Good point. The README should be updated. The important attribute to set is `coordinates`. You can (and should) omit `latitude` and `longitude`. Would welcome a PR for this.

**Comment by alexreisner** [comment]
Fixed by #1220.
